src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> converterToTransaction(List<String> inputData) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        for (String data: inputData) {
+            if (data.equals("type,fruit,quantity")) {
+                continue;
+            }

```

Comment:
Hardcoding the header string "type,fruit,quantity" is not a good practice. If the header changes in the future, you will need to change it in your code as well. Consider storing the header in a constant or configuration file, or better yet, make your code flexible enough to handle changes in the header.